admin/client/App/shared/FlashMessages.js

```diff
@@ -15,6 +15,8 @@
  */
 
 import React from 'react';
+import { isPlainObject } from 'lodash';
```

Comment:
looks good @mxstbr but remember to `import _ from 'lodash'` then use `_.isPlainObject` so we don't confuse browserify and start including two copies of lodash in the client again...